uktrade/data-hub-frontend
# Filter projects by land date instead of financial year

## Description of change

The financial year filter now works purely on land date (actual land date falling back to estimated land date) and does not use the created date at all.

## Test instructions

Go to the personalised dashboard page (enable the feature flag first) - check that the project list is filtered by land date.
Go to the investments collection list page (/investments/projects) and check that the Land date filter works correctly

## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/master/docs/Code%20review%20guidelines.md"
